src/shared/sam/cli/samCliLocalInvoke.ts

```diff
@@ -96,6 +102,7 @@ export class SamCliLocalInvokeInvocation {
         this.addArgumentIf(args, !!this.debugPort, '-d', this.debugPort!)
         this.addArgumentIf(args, !!this.dockerNetwork, '--docker-network', this.dockerNetwork!)
         this.addArgumentIf(args, !!this.skipPullImage, '--skip-pull-image')
+        this.addArgumentIf(args, !!this.debuggerPath, '--debugger-path', this.debuggerPath!)
```

Comment:
Please add tests around this arg. The other args have tests that are easy to duplicate.